cli/bin/cmdtree_dp.rs

```diff
@@ -9,6 +9,13 @@ use log::Level;
 use std::convert::AsRef;
 use strum::IntoEnumIterator;
 
+fn cmd_show_cpi() -> Node {
```

Comment:
Any reason we didn't make these Prometheus stats?  Is there a way to have hidden or second tier stats there?  I don't think we should bother changing it now, I am just curious.

Reply:
I'm not sure I'd expose those. These are not metrics that a user would be interested in. They are mainly for troubleshooting.